### ISSUE TITLE
Parser incorrectly includes comment content in $props() type transformation

### ISSUE THREAD

**Issue by chbert** [pheuter/svelte-check-rs#28]
## Description

When a comment appears on lines before `$props()`, the Rust parser incorrectly includes the comment text as part of the generated type annotation, resulting in invalid TypeScript output and hundreds of false positive errors.

## Reproduction

**Input Svelte file:**

```svelte
<script lang="ts">
  // See: some-issue-id - Description of the issue
  setServerTestFlags(pageState.data.testFeatureFlags);

  let { children } = $props();
</script>
```

**Expected transformed TypeScript:**

```typescript
// See: some-issue-id - Description of the issue
setServerTestFlags(pageState.data.testFeatureFlags);

let { children } = ({} as __SvelteLoosen<SomePropsType>);
```

**Actual transformed TypeScript:**

```typescript
let { children } = ({} as __SvelteLoosen<some-issue-id - Description of the issue
setServerTestFlags(pageState.data.testFeatureFlags);

let { children }>);
```

The comment content `some-issue-id - Description of the issue` is incorrectly being parsed as part of the type annotation.

## Impact

In a production SvelteKit codebase with ~1172 Svelte files:
- **svelte-check**: 0 errors, 0 warnings
- **svelte-check-rs**: 593 errors, 29 warnings

Most of the 593 errors are TypeScript parse errors caused by this malformed output.

## Environment

- svelte-check-rs: 0.4.1
- tsgo: 7.0.0-dev.20260106.1
- Svelte: 5.45.2
- SvelteKit: 2.48.0
- macOS 15.5 (Apple Silicon)

## Workaround

Using `--skip-tsgo` reduces errors to only 9 (actual Svelte parse errors) + 29 warnings, confirming the issue is in the Svelte→TypeScript transformation, not in tsgo itself.

## Debug output

Running with `--emit-ts --single-file` shows the malformed output clearly:

```bash
pnpm exec svelte-check-rs --emit-ts --single-file src/routes/+layout.svelte
```
